CHANGELOG.md

```diff
@@ -19,7 +21,6 @@ Breaking changes:
 
 New features:
 - Added `Foldable` and `Traversable` instances for `Graph` (#16 by @MaybeJustJames)
-- Added `toMap` to unwrap `Graph` (#18)
```

Comment:
Could you put this back into the correct spot in the changelog?

Reply:
I moved this because it hasn't been released. Would you prefer a separate PR?

Reply:
Ah, my bad. No, let's just fix it here.